### PR TITLE
update to sdk 3.0.0

### PR DESCRIPTION
**Pull request by alekseytsekov**

### CODE REVIEW

File: libraries/Ownable/test/ownable-tests.js [hunk 1]
```diff
@@ -80,11 +77,14 @@ describe('Ownable', () => {
...
 			await contractObject.call(ownableFunctions.RENOUNCE_OWNERSHIP);
 			const getOwnerResult = await contractObject.call(ownableFunctions.OWNER);
-			let defaultAddress = await getOwnerResult.decode('address');
-			
-			// !!! if address is default "#0", there is no need to convert it from hex to "normal"
-			//let address = decodedHexAddressToPublicAddress(ownerHexPublicKey);
 
+			let defaultAddress = await getOwnerResult.decode();
+
+			// until next sdk release greater than 3.0.0
```
Comment (thepiwo) [inline libraries/Ownable/test/ownable-tests.js]: why is this needed?

Comment (alekseytsekov) [reply libraries/Ownable/test/ownable-tests.js]: because current sdk do not decoded it correct

**Comment by maptuhec** [comment]
@thepiwo @mradkov  I think that we can merge it?
